iot_config/migrations/2_organizations.sql

```diff
@@ -0,0 +1,29 @@
+create type org_status as enum (
+    'enabled',
+    'disabled'
+);
+
+create table organizations (
+    oui bigserial primary key not null,
+    owner_pubkey text not null unique,
+    payer_pubkey text not null,
+    delegate_keys text[],
+    status org_status not null default 'enabled',
+
+    inserted_at timestamptz not null default now(),
+    updated_at timestamptz not null default now()
+);
+
+select trigger_updated_at('organizations');
+
+create table organization_devaddr_constraints (
+    oui bigint not null references organizations(oui) on delete cascade,
+    nwk_id int not null,
```

Comment:
Is the `nwk_id` meant to be the full `net_id`?
The same `nwk_id` can appear across the different `types` of `net_id`.

Reply:
this was just translated over from the elixir service so you tell me if this should be the full net_id or if the nwk_id would suffice? the only place we're currently using it here is in filtering db results to get the next available helium devaddr. in the elixir service it looks like it's primarily used for address parsing (which we don't need here) and later in route validation.

is it fair to say that using it for route validation is still a valuable use case or do you think we should switch our use of `nwk_id` here for `net_id` in general? would all of the devaddr constraints that make up the available helium address space share a nwk_id or would they all be different?

Reply:
I think using only `nwk_id` for route validation would be a bug. 

That would mean a Route could contain a valid devaddr range across all 7 types of NetIDs and we wouldn't necessarily be able to know which is correct.

> would all of the devaddr constraints that make up the available helium address space share a nwk_id or would they all be different?

A DevAddr contraint spanning multiple `nwk_id` would cross multiple NetIDs, which I believe should not be allowed.

Reply:
switched the use of `nwk_id` to `net_id` as the 64-bit integer version of the type